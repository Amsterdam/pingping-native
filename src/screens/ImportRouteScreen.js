import React from 'react';
import {View, StyleSheet, Dimensions} from 'react-native';
import {Content, Container, Text} from 'native-base';

const styles = StyleSheet.create({});

const ImportRouteScreen = props => {
  return (
    <Container>
      <Content>
        <Text>ImportRouteScreen</Text>
      </Content>
    </Container>
  );
};

export default ImportRouteScreen;
