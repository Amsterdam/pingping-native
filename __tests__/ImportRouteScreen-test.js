import React from 'react';
import ImportRouteScreen from '../src/screens/ImportRouteScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<ImportRouteScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
