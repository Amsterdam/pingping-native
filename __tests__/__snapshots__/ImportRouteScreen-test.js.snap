// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      <View
        noShadow={true}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderBottomColor": "transparent",
              "borderBottomWidth": 0.5,
              "elevation": 0,
              "flexDirection": "row",
              "height": 64,
              "justifyContent": "center",
              "left": 0,
              "paddingLeft": 6,
              "paddingRight": 10,
              "paddingTop": 18,
              "right": 0,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
              "top": 0,
            },
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            false,
          ]
        }
        transparent={true}
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "flex-start",
                "alignSelf": "center",
                "flex": 1,
              },
              Object {
                "flex": 1,
              },
            ]
          }
        >
          <View
            accessible={true}
            focusable={true}
            forwardedRef={[Function]}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": null,
                "backgroundColor": "transparent",
                "borderBottomWidth": null,
                "borderColor": null,
                "borderLeftWidth": null,
                "borderRadius": 5,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "borderWidth": undefined,
                "elevation": 0,
                "flexDirection": "row",
                "height": 45,
                "justifyContent": "space-between",
                "marginLeft": -3,
                "opacity": 1,
                "paddingBottom": 6,
                "paddingLeft": 4,
                "paddingRight": 6,
                "paddingTop": 6,
                "shadowColor": null,
                "shadowOffset": null,
                "shadowOpacity": null,
                "shadowRadius": null,
              }
            }
            testID="Header-Back-Button"
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "color": "#007aff",
                      "fontSize": 34,
                      "marginLeft": 1,
                      "marginRight": 2,
                      "marginTop": 0,
                      "paddingTop": 1,
                    },
                    Object {
                      "color": "#000",
                    },
                  ],
                  Object {
                    "fontFamily": "anticon",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              type="AntDesign"
            >
              î˜œ
            </Text>
          </View>
        </View>
        <Text
          style={
            Array [
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 16,
              },
              Object {
                "fontFamily": "Heavitas",
                "fontSize": 30,
                "textAlign": "left",
              },
              Object {
                "color": "#fb9f00",
                "fontSize": 14,
                "marginTop": 5,
              },
            ]
          }
          uppercase={false}
        >
          INLOGGEN
        </Text>
        <View
          style={
            Array [
              Object {
                "alignItems": "flex-end",
                "alignSelf": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "flex-end",
              },
              Object {
                "flex": 1,
              },
            ]
          }
        />
      </View>
    </RCTSafeAreaView>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": undefined,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "flex": 1,
              "padding": 25,
              "position": "relative",
              "width": 750,
            }
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000",
                  "fontFamily": "System",
                  "fontSize": 16,
                },
                Object {
                  "fontFamily": "Heavitas",
                  "fontSize": 30,
                  "textAlign": "left",
                },
                Object {
                  "marginBottom": 30,
                },
              ]
            }
            uppercase={false}
          >
            Gegevens Importeren
          </Text>
          <Text
            style={
              Object {
                "fontFamily": "Raleway-Regular",
                "fontSize": 15,
                "lineHeight": 25,
                "marginBottom": 30,
                "textAlign": "left",
              }
            }
          >
            Als je van device switcht wil je natuurlijk niet dat al jouw gegevens en prestaties op Ping Ping verloren gaan!
          </Text>
          <Text
            style={
              Object {
                "fontFamily": "Raleway-Regular",
                "fontSize": 15,
                "lineHeight": 25,
                "marginBottom": 30,
                "textAlign": "left",
              }
            }
          >
            Het is heel simpel om jouw gegevens te importeren van je oude naar je nieuwe device. Open de app op je oude device en ga naar account, klik daar op exporteer gegevens en scan de QR-code met je nieuwe device.
          </Text>
          <View
            style={
              Object {
                "alignItems": "center",
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "flex": 1,
                  },
                  Object {
                    "alignItems": "center",
                    "borderColor": "#fb9f00",
                    "borderRadius": 5,
                    "borderWidth": 3,
                    "height": 650,
                    "justifyContent": "center",
                    "overflow": "hidden",
                    "width": 650,
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "flex": 2,
                      "justifyContent": "center",
                      "width": 750,
                    },
                    undefined,
                  ]
                }
              />
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flex": 1,
                    "justifyContent": "center",
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "fontSize": 16,
                      "textAlign": "center",
                    }
                  }
                >
                  ...
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "flex": 2,
                      "justifyContent": "center",
                      "width": 750,
                    },
                    undefined,
                  ]
                }
              />
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
